streams/hk.m3u

```diff
@@ -55,3 +55,4 @@ http://zhibo.hkstv.tv/livestream/mutfysrq/playlist.m3u8
 http://221.179.217.70/PLTV/88888888/224/3221225942/1.m3u8
 #EXTINF:-1 tvg-id="" status="timeout",鳳凰衛視資訊台HD (720p) [Not 24/7]
 http://117.169.120.138:8080/live/fhzixun/.m3u8
+#EXTINF:-1 tvg-id="TVBXingHe.hk" status="",TVB Xing He 星河 (1080p)
```

Comment:
?

Reply:
Forgot to add link. Hopefully I added this time.